lib/components/user/with-logged-in-user-support.js

```diff
@@ -1,6 +1,64 @@
-import React from 'react'
+import React, { Component } from 'react'
+import { connect } from 'react-redux'
+import { withAuth } from 'use-auth0-hooks'
 
-import UserLoaderScreen from './user-loader-screen'
+import * as userActions from '../../actions/user'
+import { AUTH0_AUDIENCE, AUTH0_SCOPE } from '../../util/constants'
+import { renderChildrenWithProps } from '../../util/ui'
+import AwaitingScreen from './awaiting-screen'
+
+/**
+ * This component ensures that values under state.user are set when a user is logged in.
+ * If needed by the children, this component displays a wait screen while state.user values are being fetched.
+ * Upon completion (or if no user is logged in or if auth is disabled), it renders children.
+ */
+class UserLoaderScreen extends Component {
+  componentDidUpdate () {
```

Comment:
Sorry, this is a bit nitpicky, but could you move `withLoggedInUserSupport` above `UserLoaderScreen`?

Reply:
Addressed in https://github.com/opentripplanner/otp-react-redux/pull/167/commits/5f39e30c920d2355458ff7c4b5e90ad177d104d4